shared/featureconfig/config.go

```diff
@@ -34,9 +34,6 @@ type FeatureFlagConfig struct {
 	EnableAttestationCache   bool // EnableAttestationCache; see https://github.com/prysmaticlabs/prysm/issues/3106.
```

Comment:
You should remove what is no longer relevant from here also

Reply:
Done